src/actors/symbolication.rs

```diff
@@ -1029,12 +1032,20 @@ impl SymbolicationActor {
         self,
         request: SymbolicateStacktraces,
     ) -> Result<CompletedSymbolicationResponse, SymbolicationError> {
+        let serialize_dif_candidates = self.serialize_dif_candidates;
+
         measure_task_timeout(
             "symbolicate",
             self.do_symbolicate_impl(request),
             Duration::from_secs(3600),
         )
         .await
+        .map(|mut response| {
```

Comment:
nitpicking question, is `move |mut response|` any better for something?  I'm guessing the compiler sees through this completely and would do exactly the same for this trivial case.

Reply:
In this case it would not, since you use a copy type by value, so I think it applies the move implicitly.

Reply:
I think its more about using `self.serialize_dif_candidates` and moving self in there? unless `do_symbolicate_impl` takes ownership?